src/ontology/mondo-edit.obo

```diff
@@ -348133,20 +348135,20 @@ intersection_of: has_characteristic MONDO:0024495 ! tumor grade 1 or 2, general
 
 [Term]
 id: MONDO:0015068
-name: neuroendocrine tumor of rectum, well differentiated, low or intermediate grade
+name: rectal neuroendocrine tumor
 def: "A well differentiated, low or intermediate grade tumor with neuroendocrine differentiation that arises from the rectum." [NCIT:C135213]
 subset: gard_rare {source="MONDO:GARD"}
 subset: rare
-synonym: "neuroendocrine tumor of the rectum" RELATED [Orphanet:100081]
-synonym: "neuroendocrine tumour of the rectum" RELATED OMO:0003005 []
-synonym: "rectal neuroendocrine tumor" EXACT [NCIT:C135213, Orphanet:100081]
 synonym: "rectal neuroendocrine tumour" EXACT OMO:0003005 []
```

Comment:
this one should also be removed if you removed the synonym above.

Reply:
Comments addressed.

Reply:
@yshwetar please add MONDO:0015068 as a string (not the whole IRI)
Also, let's add NCIT:C135213 as another x-ref since it was the original name for this ncit term

Reply:
Done!